pulpcore/cli/deb/__init__.py

```diff
@@ -1,22 +1,26 @@
-from pulpcore.cli.common import main
-from pulpcore.cli.common.context import PluginRequirement
-from pulpcore.cli.common.generic import PulpCLIContext, pass_pulp_context
+from typing import Any
+
+import click
+from pulpcore.cli.common.generic import pulp_group
 
 from pulpcore.cli.deb.content import content
 from pulpcore.cli.deb.distribution import distribution
 from pulpcore.cli.deb.publication import publication
 from pulpcore.cli.deb.remote import remote
 from pulpcore.cli.deb.repository import repository
 
+__version__ = "0.0.5.dev"
+
 
-@main.group()
-@pass_pulp_context
-def deb(pulp_ctx: PulpCLIContext) -> None:
-    pulp_ctx.needs_plugin(PluginRequirement("deb"))
```

Comment:
You should move this requirement into the context classes.

Reply:
Is this the way?